### ISSUE TITLE
Cloud backup/restore for settings

### ISSUE THREAD

**Issue by mbirth** [dhowe/AdNauseam#1363]
### Describe the issue

uBlock Origin has a cloud backup/restore feature to easily sync settings between different computers. After #813 has been implemented, this would be a nice addition as well. You can probably take the code from uBlock Origin.

**Comment by dhowe** [comment]
Once we were caught up to this function in uB, lets implement/test...

**Comment by romulasry** [comment]
Sounds nice to have.

**Comment by romulasry** [comment]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Cloud-storage

**Comment by bayazidbh** [comment]
As a temporary solution, maybe importing settings can work so that it doesn't remove rules defined by the user? Changing settings isn't a big deal, but having to deal with losing rules was the reason why I eventually stopped using Cookies Auto Delete as it's a pain to keep track of.

**Comment by dhowe** [comment]
@mneunomne ?

**Comment by nyabinary** [comment]
bump :3

**Comment by dreadbalaur** [comment]
Bump. This has been a staple in uBO for a while now.

**Comment by mneunomne** [comment]
I somehow missed this back then. I will look into how bring this back to adnauseam in next release(s).

**Comment by mneunomne** [comment]
Turns out it was just the options that were not visible on the dashboard...

Fixed with https://github.com/dhowe/AdNauseam/commit/c4d6cb41c16aa9515a423146d7d31e1f408b8f5c

But unfortunatelly the collected ads can't fit in the cloud stoage restrictions though, only the filter preferences. 

Need to still thoroughly test it though.

**Comment by dreadbalaur** [comment]
Would the fitler preferences include imported filters that I subscribe to? That's probably more important than syncing the total collected ads.
